### PR TITLE
[ADD] payment_aps: add Amazon Payment Services

### PR DESCRIPTION
**Pull request by lauraschauer**
task-2802678

See also:
- https://github.com/odoo/documentation/pull/2532

### CODE REVIEW

File: addons/payment_aps/data/payment_acquirer_data.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        <field name="redirect_form_view_id" ref="redirect_form"/>
+    </record>
+
+    <record id="payment_method_aps" model="account.payment.method">
+        <field name="name">Amazon Payment Services</field>
+        <field name="code">aps</field>
+        <field name="payment_type">inbound</field>
+    </record>
+
+</odoo>
```
Comment (Feyensv) [inline addons/payment_aps/data/payment_acquirer_data.xml:15]: missing newline in the end of the file

File: addons/payment_aps/const.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Mapping of transaction states to APS payment statuses
+# See https://paymentservices-reference.payfort.com/docs/api/build/index.html#statuses
+PAYMENT_STATUS_MAPPING = {
+    'pending': ('19',),
+    'done': ('14',),
+}
```
Comment (Feyensv) [inline addons/payment_aps/const.py:8]: Shouldn't we consider more status code ? :thinking: 
Also, the doc link probably should be this one instead
https://paymentservices-reference.payfort.com/docs/api/build/index.html#transactions-response-codes

**Review by Feyensv — COMMENTED** [review]
quick curious look :duck:

**Review by AntoineVDV — APPROVED** [review]
Reviewed in pair-programming.

@robodoo override=ci/security
Reason:
- hmac: Legitimately imported and used to compare signatures.
- hashlib: Legitimately imported and used to generate signatures.

Link to ci/security build (master): https://runbot.odoo.com/runbot/batch/833555/build/18460897

@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo rebase-ff

**Pull request closed without merging**